gcn/micronautVersion.txt

```diff
@@ -1 +1 @@
-4.0.3
\ No newline at end of file
+4.0.6
```

Comment:
Is this the correct version ? should this be 4.0.7

Reply:
this is the Micronaut "core" version, the "platform" version is in  https://github.com/oracle/gcn/blob/main/gcn/gcn-core/src/main/resources/gcn-platform-versions.properties